tests/conftest.py

```diff
@@ -1349,6 +1349,16 @@ def fixture_flow_cell(bcl2fastq_flow_cell_dir: Path) -> FlowCellDirectoryData:
     )
 
 
+@pytest.fixture(name="novaseq_flow_cell_demultiplexed_with_bcl2fastq", scope="session")
+def fixture__novaseq_flow_cell_demux_with_bcl2fastq(
```

Comment:
Double underscore after `fixture`
```suggestion
def fixture_novaseq_flow_cell_demux_with_bcl2fastq(
```